crates/wasmtime/src/engine.rs

```diff
@@ -381,6 +380,7 @@ impl Engine {
             | "enable_verifier"
             | "regalloc_checker"
             | "is_pic"
+            | "unwind_info"
```

Comment:
Note: this is inconsistent with "enable_safepoints", which checks that we don't ahve the `reference_types` feature enabled. But so is "enable_simd", so I'm not sure which way to go.